Sources/Fuzzilli/Compiler/Compiler.swift

```diff
@@ -987,11 +987,53 @@ public class JavaScriptCompiler {
 
             if unaryExpression.operator == "typeof" {
                 return emit(TypeOf(), withInputs: [argument]).output
+            } else if unaryExpression.operator == "delete" {
+                let argumentExpression = unaryExpression.argument.expression!
+
+                if case .memberExpression(let memberExpression) = argumentExpression {
+                    let obj = try compileExpression(memberExpression.object)
+                    let isGuarded = memberExpression.isOptional
+
+                    if memberExpression.name != "" {
```

Comment:
Maybe `!memberExpression.isEmpty` instead

Reply:
Do you mean `!memberExpression.name.isEmpty` ? Have implemented that